lab_1/main.py

```diff
@@ -25,7 +36,13 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    if not isinstance(tokens, list):
+        return []
+    elif not isinstance(stop_words, list) or not stop_words:
+        return tokens
+    else:
```

Comment:
this return is not needed. just write `tokens1 = [i for i in tokens if i not in stop_words]
        return tokens1`